app/src/store/utils.ts

```diff
@@ -98,3 +106,71 @@ export const setTableOptionsUrlSearchParams = (tableOptions: TableOptions) => {
   setUrlSearchParams('order', order);
   setUrlSearchParams('sortBy', sortBy);
 };
+
+export const getValidTableOptionsFromUrlSearchParams = ({
```

Comment:
So I understand this correctly.  We want this function to return valid options even if passed invalid parameters.  Instead of throwing an error we are silently ignoring bad option in place of the default correct?

Reply:
Yeah exactly. If the param is invalid then it will essentially just default to the default value so as to not break the table. I assume this will rarely happen since I figure most people won't be tampering with these. The real purpose of adding this is to be able to link to specific pages, etc. so as to save the state of the table (ie for link sharing). Do you think it would be more beneficial to throw an error instead?

Reply:
I just wanted to clarify!  That totally make sense to me.   Errors for query params generally aren't a thing unless it's a page query param; sometime applications will display the 404 page for that.  But considering our 404 page is out of date, I'm happy with this

Reply:
Cool, totally agree 👍